keepalive_test.go

```diff
@@ -29,12 +28,15 @@ func TestKeepAliveTCP_Client(t *testing.T) {
 			Enable:      true,
 			WaitForPong: time.Microsecond,
 			Interval:    time.Millisecond * 100,
-			NewRetryPolicy: func() BackOff {
-				bo := backoff.NewExponentialBackOff()
-				bo.InitialInterval = 200 * time.Millisecond
-				bo.MaxInterval = 1 * time.Second
-				bo.MaxElapsedTime = 2 * time.Second
-				return bo
+			NewRetryPolicy: func() RetryFunc {
```

Comment:
Could you use `MustMakeKeepAlive` to cover it and avoid repetition?

Reply:
No because MustMakeKeepAlive  returns reasonable value but I want to simulate when keepalive fails.

Reply:
I am sure you can use `MustMakeKeepAlive` for the success case, right?

Reply:
The lastest test of keepalive used `MustMakeKeepAlive `.